api/routes/api.php

```diff
@@ -18,3 +18,17 @@
     Route::post('/tickets', 'createTicket');
 });
 
+Route::get('/test-notify', function () {
+    try {
+        Notify::sendEmail(
```

Comment:
Undefined type 'Notify'

Reply:
This is because aliases are resolved at runtime. I don't think we really need to use it as an alias so I updated it in [fix facade undefined](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5350/commits/4e9d54453c406550ccdc28d722de88b6ebd02c73)